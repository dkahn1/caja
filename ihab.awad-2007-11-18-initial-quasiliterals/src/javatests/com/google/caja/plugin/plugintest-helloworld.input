=== test:///helloworld.gxp
<gxp:template name="sayHello">
  <h1>Hello World</h1>
</gxp:template>

=== test:///helloworld.css
h1 { color: black; background: white }

=== test:///helloworld.js
function main() {
  getElementById('hello-base').setInnerHTML(sayHello());
}
