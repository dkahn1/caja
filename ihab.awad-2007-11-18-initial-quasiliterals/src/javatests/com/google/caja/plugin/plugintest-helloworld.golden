{
  var MY_TEST_PLUGIN_PRIVATE = {
    'nsPrefix': 'pre',
    'pathPrefix': '/plugin1',
    'name': 'MY_TEST_PLUGIN',
    'c1___': function sayHello() {
      var out___ = [];
      out___.push('<h1>Hello World</h1>');
      return plugin_blessHtml___(out___.join(''));
    }
  };
  var MY_TEST_PLUGIN = {
    'sayHello': MY_TEST_PLUGIN_PRIVATE.c1___
  };
  MY_TEST_PLUGIN_PRIVATE.plugin = MY_TEST_PLUGIN;
  plugin_initialize___(MY_TEST_PLUGIN_PRIVATE);
  (function () {
      MY_TEST_PLUGIN.main = function main() {
        (MY_TEST_PLUGIN.getElementById('hello-base')).setInnerHTML(MY_TEST_PLUGIN.sayHello());
      };
    }).call(MY_TEST_PLUGIN);
}

.pre h1 {
  color: black;
  background: white
}